packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -0,0 +1,204 @@
+
+// TODO verify if these are passed as props or obtained a different way
+export type ResetPasswordProps = {
+  serviceName?: string;
+  forceEmail?: string;
+  canGoBack?: boolean;
+};
+
+type FormData = {
+  email: string;
+};
+
+// eslint-disable-next-line no-empty-pattern
+const ResetPassword = ({
+  serviceName,
+  forceEmail,
+  canGoBack,
+}: ResetPasswordProps & RouteComponentProps) => {
+  usePageViewEvent('settings.reset-password', {
+    entrypoint_variation: 'react',
+  });
+
+  const [email, setEmail] = useState<string>(forceEmail || '');
+  const [emailErrorText, setEmailErrorText] = useState<string>();
+  const [isFocused, setIsFocused] = useState(false);
+  const alertBar = useAlertBar();
+  const account = useAccount();
+  const { l10n } = useLocalization();
+  const navigate = useNavigate();
+  const pageId: string = 'reset-password';
+  const onFocusMetricsEvent = 'reset-password.engage';
+
+  const { handleSubmit } = useForm<FormData>({
+    mode: 'onBlur',
+    criteriaMode: 'all',
+    defaultValues: {
+      email: '',
+    },
+  });
+
+  const onFocus = () => {
+    if (!isFocused && onFocusMetricsEvent) {
+      logViewEvent(settingsViewName, onFocusMetricsEvent, {
+        entrypoint_variation: 'react',
+      });
+      setIsFocused(true);
+    }
+  };
+
+  const navigateToConfirmPwReset = useCallback(() => {
+    // TODO: verify if passwordForgotToken and clearQueryParams needed
+    navigate(HomePath + 'confirm_reset_password', { replace: true });
+  }, [navigate]);
+
+  const onSubmit = useCallback(
+    async (email) => {
+      try {
+        logViewEvent(settingsViewName, 'reset-password.submit', {
+          entrypoint_variation: 'react',
+        });
+        // TODO password reset method
+        await account.resetPassword(email);
+        logViewEvent(settingsViewName, 'reset-password.success', {
+          entrypoint_variation: 'react',
+        });
+        navigateToConfirmPwReset();
+      } catch (e) {
+        console.log('reset-password.fail');
+        logViewEvent(settingsViewName, 'reset-password.fail', {
+          entrypoint_variation: 'react',
+        });
+        // TODO complete error handling - case 'UNKNOWN_ACCOUNT'
+        // TODO localize error message
+        // TODO pass sign up link to tooltip
+        setEmailErrorText('Unknown account');
+
+        // TODO complete error handling - case 'USER_CANCELED_LOGIN'
+        // logEvent('login.canceled')
+
+        // TODO error handling for other errors
+        alertBar.error(
+          l10n.getString(
+            'reset-password-error',
+            null,
+            'Sorry, there was a problem resetting your password'
+          )
+        );
+      }
+    },
+    [l10n, navigateToConfirmPwReset, alertBar, account]
+  );
+
+  return (
+    <>
+      <HeaderContinueTo
+        prefixDataTestId={pageId}
+        prefixFtlId={pageId}
+        headingText="Reset password"
+        serviceName={serviceName}
```

Comment:
Smol syntactic sugar, instead of `serviceName={serviceName}` we can do `{...{serviceName}}`, same thing in a couple other spots in this branch.